docs/output.md

```diff
@@ -2,40 +2,325 @@
 
-### FastQC
+### DeepARG
+
+<details markdown="1">
+<summary>Output files</summary>
+
+* `deeparg/`
+    * `db/`: contains diamond, data, database and model information
+    * `predict/`:
+        * `*.align.daa*`: Diamond alignment output.
+        * `*.align.daa.tsv`: Diamond alignment output as .tsv.
+        * `*.mapping.ARG`: contains the sequences with a probability >= --prob (0.8 default).
+        * `*.mapping.potential.ARG`: contains the sequences with a probability < --prob (0.8 default).
+
+</details>
+
+[deepARG](https://bitbucket.org/gusphdproj/deeparg-ss/src/master/) uses deep learning to characterize and annotate antibiotic resistance genes in metagenomes. It is composed of two models for two types of input: short sequence reads and gene-like sequences. In this pipeline we use the `ls` model, which is suitable for annotating full sequence genes and to discover novel antibiotic resistance genes from assembled samples. The tool `Diamond` is used as an aligner. 
+
+The `*.ARG` output files contain the following fields:
+> * ARG_NAME
+> * QUERY_START
+> * QUERY_END
+> * QUERY_ID
+> * PREDICTED_ARG_CLASS
+> * BEST_HIT_FROM_DATABASE
+> * PREDICTION_PROBABILITY
+> * ALIGNMENT_BESTHIT_IDENTITY (%)
+> * ALIGNMENT_BESTHIT_LENGTH
+> * ALIGNMENT_BESTHIT_BITSCORE
+> * ALIGNMENT_BESTHIT_EVALUE
+> * COUNTS
+
+### fARGene
+
+<details markdown="1">
+<summary>Output files</summary>
+
+* `fargene/`
+    * `fargene_analysis.log`: xxx
+    * `<sample_name>/`:
+        * `hmmsearchresults/`: xxx.
+        * `predictedGenes/`: 
+            * `*-filtered.fasta`: nucleotide sequences of predicted ARGs.
+            * `*-filtered-peptides.fasta`: aminoacid sequences of predicted ARGs.
+        * `results_summary.txt`: xxx.
+        * `tmpdir/`: xxx
+
+</details>
+
+[fARGene](https://github.com/fannyhb/fargene) (Fragmented Antibiotic Resistance Gene Identifier) is a tool that takes either fragmented metagenomic data or longer sequences as input and predicts and delivers full-length antiobiotic resistance genes as output. The tool includes developed and optimized models for a number or resistance gene types. The model to be used has to be specified for each run (`--hmm-model`). Models available are:
+> * `class_a`: Class A beta-lactamases
+> * `class_b_1_2`: Subclass B1 and B2 beta-lactamases
+> * `class_b3`: Subclass B3 beta-lactamases
+> * `class_d_1`: Class C beta-lactamases
+> * `class_d_2`: Class D beta-lactamases
+> * `qnr`: Quinolone resistance genes
+> * `tet_efflux`, `tet_rpg`, `tet_enzyme`: Tetracycline resistance genes
+
+<!--### RGI
+
+<details markdown="1">
+<summary>Output files</summary>
+
+* `rgi/`
+    * `output1`: xxx
+    * `output2/`: xxx
+
+</details>
+
+[RGI](https://github.com/arpcard/rgi) (Resistance Gene Identifier) predicts resistome(s) from protein or nucleotide data based on homology and SNP models. It uses reference data from the Comprehensive Antibiotic Resistance Database (CARD).
+-->
+
+### hAMRonization
```

Comment:
I would maybe put this section near the end, or even at teh top with the other summary file tools. It reports the output of all the tools, so I don't think it makes sense to put it in the middle of the tools.

Reply:
I put it after the ARG tools, because it will summarise them. The other summariser, 'combiAMP' would then follow after the AMP tools. And then there should be another one for BGCs, following 'antiSMASH'. But of course we can put all the summaries at the end of the tool description, maybe before MultiQC?